### DISCUSSION TITLE
Launch .earth file problem

### DISCUSSION THREAD

**Discussion by marcoma9023** [pelicanmapping/osgearth#1865] [Support Q&A] [answered]
launching from windows command prompt eg osgearth_viewer.exe triton.earth, I obtain the following logs:

Error reading file triton.earth: file not handled
[osgEarth]* [MapNodeHelper] Loaded scene graph does not contain a MapNode - aborting
[osgEarth]
Usage: osgearth_viewer.exe file.earth
  --sky                         : add a sky model
  --kml <file.kml>              : load a KML or KMZ file
  --kmlui                       : display a UI for toggling nodes loaded with --kml
  --coords                      : display map coords under mouse
  --ortho                       : use an orthographic camera
  --logdepth                    : activates the logarithmic depth buffer
  --logdepth2                   : activates logarithmic depth buffer with per-fragment interpolation
  --shadows                     : activates model layer shadows
  --out-earth [file]            : write the loaded map to an earth file
  --uniform [name] [min] [max]  : create a uniform controller with min/max values
  --define [name]               : install a shader #define
  --path [file]                 : load and playback an animation path
  --extension [name]            : loads a named extension
  --ocean                       : add a simple ocean model (requires bathymetry)

or if I try to launch with

osgearth_toc.exe triton.earth, the application is correctly started, but I can't see the water...

How can I run triton.earth file?

**Comment by plevy** [comment]
Let me ask the most basic question, you do have Triton installed and no errors in cmake or visual studio?  By default, the nodekit is optional and disabled in cmake.

**Reply by marcoma9023** [reply]
Triton it's installed and integrated into my application...

I don't have errors in cmake and visual studio...

**Comment by marcoma9023** [comment, marked as answer]
OK, 

the problem is solved, the .exe files needed the dll and lib...
